Add the Compatibility Schema of IntelliJ (#1490)
ballerina-platform/ballerina-www: website/two-column-pages/docs/learn/tools-ides/intellij-plugin.md

- [Using the plugin](#using-the-plugin)
- [Using the features of the plugin](#using-the-features-of-the-plugin)

> ## Downloading IntelliJ IDEA

Reviewer: ```suggestion
## Prerequisites
```

Author: Fixed.

> Download the [IntelliJ IDEA editor](https://www.jetbrains.com/idea/download/) (or update it) by selecting the version from the below table based on the compatibility with the Ballerina plugin version, which you will be using.

Reviewer: ```suggestion
Please refer the table below and [Download IntelliJ IDE](https://www.jetbrains.com/idea/download/) (or update the exiting IDE) based on its compatibility with the  plugin version (which is same as the Ballerina distribution version), which you will be using.

```

Author: Discussed and changed.

**Plugin Version**|**Platform Version Compatibility**
:-----:|:-----:
0.8.0 - 0.8.2|IntelliJ IDEA 2016.3 - 2016.4
0.8.3 - 0.981.0|IntelliJ IDEA 2016.3+

> 0.982.0+|IntelliJ IDEA 2017.3+

Reviewer: ```suggestion
0.982.0 - 0.991.0|IntelliJ IDEA 2017.3+
0.991.1+ | IntelliJ IDEA 2018.3+
```

Author: Fixed.
